File: models/category_definition.py
from tortoise import fields, models
from tortoise.models import Model


class CategoryDefinition(Model):

    id = fields.IntField(pk=True)
    symbol = fields.TextField()
    year = fields.IntField()
    description = fields.TextField()
    factor = fields.DecimalField(3,3)

    def __str__(self):
        return self.symbol

File: better_api/KutnoAPI/KutnoAPI/Models/CategoryDefinition.cs
using System.Text.Json.Serialization;

namespace KutnoAPI.Models;

public class CategoryDefinition
{
    [JsonPropertyName("symbol")]
    public string Symbol { get; set; }

    [JsonPropertyName("year")]
    public int Year { get; set; }

    [JsonPropertyName("description")]
    public string Description { get; set; }

    [JsonPropertyName("factor")]
    public decimal Factor { get; set; }
}
